src/App/Pages/Accounts/TwoFactorPage.xaml.cs

```diff
@@ -45,25 +47,11 @@ public TwoFactorPage(bool? authingWithSso = false, AppOptions appOptions = null,
             {
                 ToolbarItems.Remove(_cancelItem);
             }
-        }
-
-        public HybridWebView DuoWebView { get; set; }
 
-        public void AddContinueButton()
-        {
-            if (!ToolbarItems.Contains(_continueItem))
-            {
-                ToolbarItems.Add(_continueItem);
-            }
+            ContinueButton = _continue;
```

Comment:
You don't need this anymore after the last round of changes.

Reply:
✅